### ISSUE TITLE
tts: Increase TTS volume to 0 DB.

### ISSUE THREAD

**Issue by domasofan** [k2-fsa/sherpa-onnx#851]
Hi all,

This seems to affect tts in general.
Tested on Android 14 and Windows 11.
the generated speech seems to be very quiet and it seems to be under 0 db.
According to my normalizer it was at -8.31 DB using the de_DE-thorsten-high model.

I don't know if it is the model or if it is the tts engine.
If it is the tts engine would it be possible to raise the volume to a louder volume so it can be understood well?

Greetings and thanks,
Simon

**Comment by malnvenshorn** [comment]
I agree, the voice is very quiet. A volume slider next to the speed slider would be nice, e.g. to adjust the volume from 0% to 200%.

**Comment by domasofan** [comment]
And also a command line argument for the cli users.
Prevering the cli here because of much more versatility.
